application/language/spanish/station_lang.php

```diff
@@ -46,72 +46,72 @@
-$lang['station_location_wwff_hint_ln1'] = "Station WWFF reference. You can look up WWFF references at the <a target='_blank' href='https://www.cqgma.org/mvs/'>GMA Map</a> website.";
-$lang['station_location_pota_hint_ln1'] = "Station POTA reference. You can look up POTA references at the <a target='_blank' href='https://pota.app/#/map/'>POTA Map</a> website.";
-$lang['station_location_signature'] = "Signature";
-$lang['station_location_signature_name'] = "Signature Name";
-$lang['station_location_signature_name_hint'] = "Station Signature (e.g. GMA)..";
-$lang['station_location_signature_info'] = "Signature Information";
-$lang['station_location_signature_info_hint'] = "Station Signature Info (e.g. DA/NW-357).";
-$lang['station_location_eqsl_hint'] = 'The QTH Nickname which is configured in your eQSL Profile';
-$lang['station_location_eqsl_defaultqslmsg'] = "Default QSLMSG";
-$lang['station_location_eqsl_defaultqslmsg_hint'] = "Define a default message that will be populated and sent for each QSO for this station location.";
-$lang['station_location_qrz_subscription'] = 'Subscription Required';
-$lang['station_location_qrz_hint'] = "Find your API key on <a href='https://logbook.qrz.com/logbook' target='_blank'>the QRZ.com Logbook settings page";
-$lang['station_location_qrz_realtime_upload'] = 'QRZ.com Logbook Realtime Upload';
-$lang['station_location_hrdlog_username'] = "HRDLog.net Username";
-$lang['station_location_hrdlog_username_hint'] = "The username you are registered with at HRDlog.net (usually your callsign).";
-$lang['station_location_hrdlog_code'] = "HRDLog.net API Key";
-$lang['station_location_hrdlog_realtime_upload'] = "HRDLog.net Logbook Realtime Upload";
-$lang['station_location_hrdlog_code_hint'] = "Create your API Code on <a href='http://www.hrdlog.net/EditUser.aspx' target='_blank'>HRDLog.net Userprofile page";
-$lang['station_location_qo100_hint'] = "Create your API key on <a href='https://qo100dx.club' target='_blank'>your QO-100 Dx Club's profile page";
-$lang['station_location_qo100_realtime_upload'] = "QO-100 Dx Club Realtime Upload";
-$lang['station_location_oqrs_enabled'] = "OQRS Enabled";
-$lang['station_location_oqrs_email_alert'] = "OQRS Email alert";
-$lang['station_location_oqrs_email_hint'] = "Make sure email is set up under admin and global options.";
-$lang['station_location_oqrs_text'] = "OQRS Text";
-$lang['station_location_oqrs_text_hint'] = "Some info you want to add regarding QSL'ing.";
-$lang['station_location_clublog_realtime_upload']='ClubLog Realtime Upload';
+$lang['station_location'] = 'Localización de Estación';
+$lang['station_location_plural'] = "Localizaciones de Estaciones";
+$lang['station_location_header_ln1'] = 'Las localizaciones de Estación le permiten definir localizaciones de operación, como su QTH, el QTH de un amigo o una estación portátil.';
```

Comment:
"Portable" rather than "portátil". That is how Spanish people always call them in ham radio.